### DISCUSSION TITLE
RSunda

### DISCUSSION THREAD

**Discussion by RakeshSunda** [community/community#140412] [Programming Help]
### Select Topic Area

Show & Tell

### Body

try {
  // no need to wait 30sec 'ok-selector' in case of dead_page()
  wait('ok-selector');
} catch(e) {
  // in this case we can't be sure that the page is real dead
  dead_page('Page doesn\'t exist');
[}``](rsunda7.slack.com)

**Comment by ebndev** [comment]
Hi @RakeshSunda, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share more information about your project / code snippets or a link to your repo. Giving a few more details might help other users better engage with your discussion 😄
